.gitignore

```diff
@@ -26,7 +26,6 @@ envfile
 # kubeconfigs
 kind.kubeconfig
 minikube.kubeconfig
-kubeconfig
```

Comment:
We're still generating `kubeconfig` in the Makefile, and need to have it ignored. Change it to `/kubeconfig` which ignores the file on top-level, but not in subdirectories.

Reply:
Done, thanks